website/docs/components/tabs/partials/guidelines/guidelines.md

```diff
@@ -8,6 +8,46 @@
 
 - For navigation, consider navigation, [Standalone Links](/components/link/standalone), or [Breadcrumb](/components/breadcrumb).
 
+### Usage examples
+
+!!! Do
+
+Use Tabs to break up content that is related to each other and is on the same hierarchical level.
+
+![Tab usage example](/assets/components/tabs/tab-usage-example-01.png)
+!!!
+
+!!! Dont
+
+Don't use Tabs to take the user to a new URL or as a primary or secondary navigation.
```

Comment:
FWIW, a query param is technically a new URL, and that's how we use our tabs component in our own docs (i.e., https://helios.hashicorp.design/components/modal?tab=code) 

I wonder if there's a different way to word this?

Reply:
While the Helios website tabs and the HDS tabs differ, I addressed this in another round of iterations.